### DISCUSSION TITLE
Request to add Chinese translation

### DISCUSSION THREAD

**Discussion by Gzzloveyyt** [music-assistant/support#1318] [Feature requests and ideas]
Hello author, I am Chinese and have been using the old version of MUSIC ASSISTANT since then. At that time, my could modify the JS file to translate it yourself. Now that you have replaced it with a brand new music assistant, I don't know how to translate it. Could you please adapt to Chinese and choose your own language in the settings?

**Comment by OzGav** [comment]
We are looking for help with translations. If you can then open an issue and use the “ Translation Request Music Assistant.” template and someone will,get back to you

**Comment by jozefKruszynski** [comment]
@gzzyyt If you would like to contribute a language, then please do as @OzGav mentioned.
